### PR TITLE
remove TODO from Profiles section

### PR DESCRIPTION
**Pull request by kwsutter**
Signed-off-by: Kevin Sutter <kwsutter@gmail.com>

Since CORBA and IIOP are still Optional for Jakarta EE 9, this TODO can be removed.

### CODE REVIEW

**Review by dblevins — APPROVED** [review]
For those reading, we will remove the `java:comp/ORB` JNDI name in 9.1

**Comment by kwsutter** [comment]
Thanks, @dblevins, for the quick review and approval.  Will wait until tomorrow to merge to allow others to possibly review.

**Review by scottmarlow — APPROVED** [review]

**Comment by scottmarlow** [comment]
`RMI/IIOP interoperability requirement` is marked as optional which is wrong as per release plan `Support for Distributed Interoperability in the EJB 3.2 Core Specification, Chapter 10`

**Review by scottmarlow — COMMENTED** [review]
Looks fine to me once the `interop` is removed from the list of optional technologies (note that `interop` is correctly shown at the bottom of page as removed).

**Comment by kwsutter** [comment]
> `RMI/IIOP interoperability requirement` is marked as optional which is wrong as per release plan `Support for Distributed Interoperability in the EJB 3.2 Core Specification, Chapter 10`

Distributed Interop was removed.  But, the use of RMI/IIOP may still be used by some implementations for their own interop capabilities.  For Jakarta EE 9, I think it's okay to say that this is "optional" (as it was in Java EE 8 and Jakarta EE 8).

**Comment by kwsutter** [comment]
> Looks fine to me once the `interop` is removed from the list of optional technologies (note that `interop` is correctly shown at the bottom of page as removed).

@scottmarlow Based on my previous comment, are you okay with this moving forward as-is?  Thanks!

**Comment by scottmarlow** [comment]
> > Looks fine to me once the `interop` is removed from the list of optional technologies (note that `interop` is correctly shown at the bottom of page as removed).
> 
> @scottmarlow Based on my previous comment, are you okay with this moving forward as-is? Thanks!
> This is consistent with [my other PR](https://github.com/eclipse-ee4j/jakartaee-platform/pull/256) to clean up the PlatformOverview section.

I didn't expect that `interop` was changing from `removed` to `optional`.  I only expected that the Enterprise Beans 4.0 SPEC will still expect `javax.rmi.PortableRemoteObject` + `org.omg.CORBA.ORB` classes to be be available and work as expected (to be addressed after 9.0).

`Interop` is a promise that each Jakarta EE 9 server implementation will be capable of interoperating with at least one other server implementation.  This would be an untested promise to make at the Platform level since we removed the `interop` tests before we could run the Platform TCK against GlassFish 6.0.

**Comment by kwsutter** [comment]
> > > Looks fine to me once the `interop` is removed from the list of optional technologies (note that `interop` is correctly shown at the bottom of page as removed).
> > 
> > 
> > @scottmarlow Based on my previous comment, are you okay with this moving forward as-is? Thanks!
> > This is consistent with [my other PR](https://github.com/eclipse-ee4j/jakartaee-platform/pull/256) to clean up the PlatformOverview section.
> 
> I didn't expect that `interop` was changing from `removed` to `optional`. I only expected that the Enterprise Beans 4.0 SPEC will still expect `javax.rmi.PortableRemoteObject` + `org.omg.CORBA.ORB` classes to be be available and work as expected (to be addressed after 9.0).
> 
> `Interop` is a promise that each Jakarta EE 9 server implementation will be capable of interoperating with at least one other server implementation. This would be an untested promise to make at the Platform level since we removed the `interop` tests before we could run the Platform TCK against GlassFish 6.0.

Okay, now I understand the confusion.  So, how about if we change this bullet from `RMI/IIOP interoperability requirements` to `CORBA requirements`?  Since this was referencing the OMG Protocol section, that seems more consistent.  We can avoid the RMI/IIOP topic altogether in this section.  Would that help clarify?

**Pull request merged**
